src/plugins/telemetry_management_section/public/components/telemetry_management_section.tsx

```diff
@@ -113,7 +130,10 @@ export class TelemetryManagementSection extends Component<Props, State> {
             <EuiSplitPanel.Inner color="subdued">
               <EuiTitle>
                 <h2>
-                  <FormattedMessage id="telemetry.usageDataTitle" defaultMessage="Usage Data" />
+                  <FormattedMessage
+                    id="telemetry.usageDataTitle"
+                    defaultMessage="Usage Collection"
```

Comment:
Sentence case?

Usage collection

Reply:
I was following similar casing convention as its parent menus `Stack Management`, `Kibana`, `Global Settings`. But I agree that the correct form is to lower case collection :)